src/models/read-receipt.ts

```diff
@@ -142,8 +142,7 @@ export abstract class ReadReceipt<
             //    was deleted.)
             //
             // 4. The receipt had the incorrect thread ID (due to a bug in a
-            // client, or malicious behaviour).
-            logger.warn(`Ignoring receipt for missing event with id ${receipt.eventId}`);
```

Comment:
This only just got added - cc @andybalaam

Reply:
Yes. It's also currently making a test fail, but I'll leave fixing that up until we decide what to do.

EW currently produces a firehose of these warnings, so I'm struggling to believe they are useful.

Reply:
This usually indicates a bug, unless we got unlucky with a message pointed to by a receipt being deleted.

@florianduros and I have a PR we are hoping to get ready today that fixes 2 bugs that would have caused this message, so please can we tolerate it a little longer until that PR can get merged?

Reply:
Have removed this bit of the PR.

Reply:
this got added back again in https://github.com/matrix-org/matrix-js-sdk/pull/3904